src/platform_impl/mod.rs

```diff
@@ -58,15 +58,12 @@ impl From<Fullscreen> for RootFullscreen {
 }
 
 #[cfg(all(
-    not(target_os = "ios"),
-    not(target_os = "windows"),
-    not(target_os = "linux"),
+    not(x11),
+    not(wayland),
```

Comment:
```suggestion
    not(linux),
```
Let's keep things 1:1. There's a separate error when you don't enable either the `x11` or `wayland` features.

Reply:
sure but I will have to switch it to be `target_os = "linux"` because of clippy